dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/WithSpanServlet.java

```diff
@@ -0,0 +1,115 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.not;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.instrumentation.annotations.SpanAttribute;
+import io.opentelemetry.instrumentation.annotations.WithSpan;
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.servlet.annotation.WebServlet;
+
+@SuppressWarnings("serial")
+@WebServlet("/WithSpanServlet")
+public class WithSpanServlet extends FATServlet {
+
+    @Inject
+    private SpanBean spanBean;
+
+    private static final String PARAMETER_TEST = "testString";
+
+    @Test
+    public void callNotAnnotated() {
+        //Returned spanId from methodNotAnnotated should not be the spanID of 0000000000000000
+        //No span was created
+        String spanId = spanBean.methodNotAnnotated();
+        assertThat(spanId, equalTo("0000000000000000"));
```

Comment:
In these tests where we're trying to make sure that a span is created or not created by a method call, I think it would be better to capture the current span at the start of the method and compare the other spanId with that.

E.g.
```java
Span originalSpan = Span.current();

String newSpanId = spanBean.methodNotAnnotated();
assertThat(newSpanId, equalTo(originalSpan.getSpanContext().getSpanId());
```